server/app/assets/stylesheets/styles.css

```diff
@@ -43,6 +43,11 @@
     color: #005ea2;
   }
 
+  .usa-button.usa-modal__close {
+    background-color: transparent;
+    color: #71767a;
```

Comment:
You don't need to do it in this PR. When you have a change can you move all the hex colors into sass variables.

Reply:
will do! filed https://github.com/civiform/civiform/issues/7592